src/index.html

```diff
@@ -7,12 +7,77 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        />
+      </div>
+
+      <div class="card__title">
+        <p class="card__title-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__title-product-code">
+          Product code:
+          <span>195434</span>
+        </p>
+      </div>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__review-text">
+          Reviews:
+          <span>5</span>
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-name">Price:</p>
+
+        <p class="card__price-amount">$2,199</p>
+      </div>
+
+      <button
```

Comment:
**there 2 BEM blocks card and stars**  (no `main-button`)
so rename it to card__main-button